### ISSUE TITLE
Rails 5: deprecation warning regarding the use of uniq

### ISSUE THREAD

**Issue by TheCount** [dwbutler/groupify#52]
```
ActiveSupport::DeprecationException:
DEPRECATION WARNING: uniq is deprecated and will be removed from Rails 5.1 (use distinct instead) (called from […])
# /var/lib/gems/2.3.0/gems/groupify-0.8.0/lib/groupify/adapter/active_record/group_member.rb:25:in `block (2 levels) in <module:GroupMember>'
[…]
```

This (group_member.rb:25) is the only place the warning was triggered in my case, but a cursory search over the groupify source code indicates `uniq` is used in other places as well.

**Comment by dwbutler** [comment]
I noticed that as well. I tried changing `uniq` to `distinct` as suggested, but this broke compatibility with Rails 3.2. I think the cleanest thing to do will be to stop supporting Rails 3.2 once Rails 5.1 is released.

**Comment by TheCount** [comment]
@dwbutler Fair enough

**Comment by rposborne** [comment]
@dwbutler I through together a pull request that implements what you have suggested, travis is blowing up over pry... which seems odd, and unrelated.

**Comment by dwbutler** [comment]
@rposborne I resolved some dependency issues. It happens - the world of open source is constantly changing around us, even if your own code hasn't changed.

Let's continue the conversation in #53.

**Comment by dwbutler** [comment]
Resolved in 0.9.0.
